tests/5.1/error/test_error_at_runtime.c

```diff
@@ -0,0 +1,35 @@
+//===--- test_error_at_runtime.c ----------------------------===//
+//
+//  OpenMP API Version 5.1 Aug 2021
+//
+//  Test uses error directive with at(error) clause to print a message,
+//  if error condition is reached during runtime, where it will print a,
+//  error message. If error message does not print, then the test has failed. 
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+int errors, i;
+
+int test_error_at_runtime() {
+    i = 0;
+    printf("Test should print a 'runtime' error message: \n");
+    #pragma omp parallel
+    {
+        #pragma omp error at(runtime)
```

Comment:
This fails with:
```
test_error_at_runtime.c:24:30: error: expected ‘execution’ or ‘compilation’
   24 |         #pragma omp error at(runtime)
```
I think the keyword was changed late in the process – and the example document got it wrong. (Corrected for the next iteration but still to late ...)

BTW: With GCC (12 or devel version), the program stops at this directive with exit code 1 and the output:
```
libgomp: 
libgomp: fatal error: error directive encountered
```